### PR TITLE
PIM-3984: publish process

### PR DESCRIPTION
**Pull request by sumbobyboys**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/associations.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ define(
                     }, this));
                 }, this));
 
-                mediator.on('post_save', _.bind(function () {
+                mediator.on('product:action:post_update', _.bind(function () {
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/associations.js]: :+1:

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes.js [hunk 2]
```diff
@@ -45,14 +47,14 @@ define(
             configure: function () {
                 this.getRoot().addTab('attributes', 'Attributes');
                 this.state = new Backbone.Model({
-                    'locale': 'en_US',
-                    'scope':  'ecommerce'
+                    'locale': usercontext.getUserContext().catalog_locale,
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes.js]: not .get('catalogLocale')?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/meta/created.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+ 'use strict';
+
+define(
+    [
+        'jquery',
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/meta/created.js]: unused dep

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes.js (hunk 2 above)
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes.js]: yeah, not sure about this one. We could

**Pull request merged**